assets/js/query/boolean.ts

```diff
@@ -0,0 +1,35 @@
+import { AstMatcher } from './types';
+
+export function matchAny(...matchers: AstMatcher[]): AstMatcher {
+  return (e: HTMLElement) => {
+    for (const matcher of matchers) {
+      if (matcher(e)) {
+        return true;
+      }
+    }
+    return false;
+  };
```

Comment:
Similarly this can be simplified to `return matchers.some(matcher => matcher(e));`